app-web/src/components/Cards/Card/index.js

```diff
@@ -50,6 +50,7 @@ export const CardWrapper = styled.article`
   box-shadow: 0 1px 2px 1px #00000026;
   display: flex;
   flex-flow: column nowrap;
+  overflow: hidden;
```

Comment:
for cards with extra long text, does this just cut the text off or can you scroll?